### ISSUE TITLE
Starting Client Run Configuration Instead Of Server

### ISSUE THREAD

**Issue by khalidabuhakmeh** [MartinLahucky/BlazorProblem#1]
👋 Hello Martin,

This is Khalid Abuhakmeh, your friendly neighborhood JetBrains Advocate. 

TL;DR: You need to start the **BlazorProblem.Server** run configuration, not the BlazorProblem.Client run configuration. Explanation below.

---

You're running into a common usability issue that I will bring up with devs on Rider, but I'll explain the problem here first.

When starting a Blazor Hosted solution, you have three projects:

- Client
- Server
- Shared

Rider sees two run configurations from this solution structure: Client and Server.

![image](https://user-images.githubusercontent.com/228256/217256505-bfb48ca3-4e10-43b9-838a-3d19813e126d.png)

If you start the **Client** project, you'll only get the Blazor WASM side of your project. This is **NOT** what most folks want. 

If you start the **Server** project, you'll get both the backend and front end of your application. This is what folks want.

When the server is started, you'll be able to hit the Kestrel-hosted API you've created.

![image](https://user-images.githubusercontent.com/228256/217257100-8b86ffc7-8ab3-417a-b05e-2fbe105e9cde.png)

Cheers, 

Khalid :)

**Comment by MartinLahucky** [comment]
Thank you so much. Sorry to have trouble with this issue, and I didn't realize that this could be the cause.

**Comment by khalidabuhakmeh** [comment]
No worries. Cheers. 🍻
